src/cms-content/team/alumni.json

```diff
@@ -383,6 +383,16 @@
       "profileImgUrl": "/images_cms/bar_recommends_icon.svg",
       "fullName": "Marta Wosińska",
       "profileUrl": "https://healthpolicy.duke.edu/people/marta-wosinska-phd"
+    },
+    {
+      "profileImgUrl": "/images_cms/bar_recommends_icon.svg",
```

Comment:
Heh.  This looks wrong, but I guess it's what we use for all of them. 🤷

Reply:
hahaha yes this is a relic from fixing an error a while ago. everyone defaults to a cocktail emoji. i should change this